response.go

```diff
@@ -47,34 +50,40 @@ func (body *Body) Payload() []byte {
 			return nil
 		}
 
-		// XXX: Handle this error
-		bytes, _ := ioutil.ReadAll(body.File)
+		bytes, err := ioutil.ReadAll(body.File)
+		if err != nil {
+			return []byte(fmt.Sprintf("File could not be read: %s \n", body.File.Name()))
+		}
 		body.File.Seek(0, 0)
 		return bytes
 	}
-	return nil
+	return []byte("No body configured.")
 }
 
+// Info returns some basic info on the body.
 func (body *Body) Info() []byte {
 	switch body.Mode {
 	case BodyInput:
-		return body.Input
+		return []byte(fmt.Sprintf("size: %d bytes\n", len(body.Input)))
```

Comment:
This is making return not the user's body input but `size: %d bytes`

Reply:
Yes, i thought that's what Info() should do. Returning meta-infos on the body, rather than returning the actual body. Analogous to what it does with file-based bodies.

Reply:
Actually, when Body mode is BodyInput, Info should be nil, Info refers to the file, the coupling it's kind of ugly, I agree.
But if we merge that, we're introducing a bug. So I can't merge it

Reply:
Alright, there might be some parts i don't know about that depend on this behavior. You should be able to go ahead and merge ;)